tests/rados/test_scrub_omap.py

```diff
@@ -0,0 +1,117 @@
+"""
+This file contains the  methods to verify the  OSD status during large omap entries.
+AS part of verification the script  perform the following tasks-
+   1. Creating omaps
+   2. Perform scrub and deepscrub
+   3. While creating objects check that osds status
+"""
+
+
+import datetime
+import time
+from threading import Thread
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.pool_workflows import PoolFunctions
+from utility.log import Log
+from utility.utils import method_should_succeed
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Test to create a large number of omap entries on the single PG pool and test osd resiliency
+    Returns:
+        1 -> Fail, 0 -> Pass
+    """
+    log.info(run.__doc__)
+    config = kw["config"]
+
+    global rados_obj
```

Comment:
Can we avoid Global variables ? I do not believe we need these global variables used here to pass data among methods. 

This would create lot of issues in future when we import this module for future automations. 

I would suggest to remove these. Let me know what you think.

Reply:
Removed the global variables in the code.  Refer https://github.com/red-hat-storage/cephci/pull/2340